build/scripts-3.9/error_check_rabies.py

```
@@ -0,0 +1,245 @@
+    import sys
+    command = opts.custom
+    if 'preprocess' in command:
+        command += f" --anat_inho_cor method=disable,otsu_thresh=2,multiotsu=false --bold_inho_cor method=disable,otsu_thresh=2,multiotsu=false \
+    --anat_template {tmppath}/inputs/sub-token1_T1w.nii.gz --brain_mask {tmppath}/inputs/token_mask.nii.gz --WM_mask {tmppath}/inputs/token_mask.nii.gz --CSF_mask {tmppath}/inputs/token_mask.nii.gz --vascular_mask {tmppath}/inputs/token_mask.nii.gz --labels {tmppath}/inputs/token_mask.nii.gz \
+    --bold2anat_coreg registration=no_reg,masking=false,brain_extraction=false,keep_mask_after_extract=false,winsorize_lower_bound=0.005,winsorize_upper_bound=0.995 --commonspace_reg masking=false,brain_extraction=false,keep_mask_after_extract=false,fast_commonspace=true,template_registration=no_reg,winsorize_lower_bound=0.005,winsorize_upper_bound=0.995 --data_type int16"
+        command += f" {tmppath}/inputs {tmppath}/outputs"
+        
+    if 'confound_correction' in command or 'analysis' in command:
+        if not os.path.isfile(f'{tmppath}/outputs/rabies_preprocess_workflow.pkl'):
+            # provide preprocess outputs to run cc stage
+            process = subprocess.run(
```

Ruff (0.14.1)

`subprocess` call with `shell=True` identified, security issue

(S602)